doc/sphinx/user_manual/basic_starter_tutorial/14_different_models_in_segments.md

```diff
@@ -0,0 +1,65 @@
+(part:user_manual:chap:basic_starter_tutorial:sec:14_different_models_in_segment)=
+Different models in segments
+============================
+
+Slabs and faults can be heterogeneous in depth, and this is very easy to represent in the world builder. The segments we just added allow for defining temperature and compositional models, which overwrite the temperature and compositional models for the whole feature. You can think of it this way: you set the default models for each segment in the feature (line 54 and 55) and then you overwrite that default model in the segments where you want something else. 
```

Comment:
I see what you mean, but I don't really like the term general models. I will think about how to reword this.

Reply:
Yeah I agree general isn't great either.  I was having a hard time finding the right word for this since general didn't feel correct either. I'm not sure if "overall" or "broad", "universal" is better?

Reply:
what about "feature" model? Since it is the the (temperature/composition) model of the feature which is actually set as the "default" which you can overwrite.

Reply:
Yeah I think that makes sense!